config/locales/en.yml

```diff
@@ -110,14 +99,16 @@ en:
         link_to_setting_type_other: I cannot find my setting or organisation
         autocomplete-not-found: No setting found.
       update:
-        complete: Thank you for creating a child development training account. You can now start the first module.
+        complete_registration: Thank you for creating a child development training account. You can now start the first module.
```

Comment:
@ebrett rather than defining the same keys four times could we instead float these locales up, just under `registration:` so they can be reused?

Reply:
I was going with the default scopes used by rails - plus, making it easier if they do want different messages at a later time.  But I agree, probably should be more DRY.

Reply:
I'd like to save that for later though, so can you approve now so we can get this into staging?